common/low_pass_filter.c

```diff
@@ -1,5 +1,20 @@
 #include "low_pass_filter.h"
 
-status_t low_pass_filter_init(void) {
-    return W_FAILURE;
+static volatile double alpha_low;
```

Comment:
We need more than one low pass filter per firmware, so the user of the library need to have this as a pointer to pass to the low pass filter functions